lib/src/base-styles/_nx-colors.scss

```diff
@@ -40,6 +40,7 @@
   --nx-color-threat-moderate: var(--nx-swatch-yellow-60);
   --nx-color-threat-severe: var(--nx-swatch-orange-50);
   --nx-color-threat-critical: var(--nx-swatch-red-40);
+  --nx-color-threat-zero: var(--nx-swatch-grey-90);
```

Comment:
This isn't really a threat color but rather is a special styling treatment specific to `NxSmallThreatCounter`.  So I don't think a CSS variable should be added for it.  In particular it could easily be misconstrued: the other variables here are about different threat severity levels, while the zero-styling could be applied to a counter at any severity level and is about not the level, but the count of violations at that level.

Reply:
> This isn't really a threat color but rather is a special styling treatment specific to `NxSmallThreatCounter`. So I don't think a CSS variable should be added for it. In particular it could easily be misconstrued: the other variables here are about different threat severity levels, while the zero-styling could be applied to a counter at any severity level and is about not the level, but the count of violations at that level.

That makes sense. It has been removed here: [6a16448](https://github.com/sonatype/sonatype-react-shared-components/pull/740/commits/6a16448d850c8714a20c661aa2f7b4b8dde9ce89)